manifests/0000_50_cluster-storage-operator_01_operator_namespace.yaml

```diff
@@ -1,3 +1,4 @@
+# Create the namespace at runlevel 49, so cluster-csi-snapshot-controller-operator (running at level 50) can use it.
 apiVersion: v1
 kind: Namespace
 metadata:
```

Comment:
Was [the `openshift.io/cluster-monitoring: "true"` label][1] important?

[1]: https://github.com/openshift/cluster-csi-snapshot-controller-operator/pull/49/files#diff-c90f942468bb6270eaab50e76232634eL8

Reply:
Restored